functions/src/schedule-generator/speakers-sessions-schedule-map.js

```diff
@@ -0,0 +1,186 @@
+function sessionsSpeakersScheduleMap(sessionsRaw, scheduleRaw, speakersRaw) {
+    let schedule = {};
+    let sessions = {};
+    let speakers = {};
+    let scheduleTags = [];
+
+    for (const dayKey of Object.keys(scheduleRaw)) {
```

Comment:
use `forEach` as you did previously to be consistent :)

Reply:
but if we care about performance, classic `for` is better
https://medium.com/tech-tajawal/loops-performances-in-node-js-9fbccf2d6aa6

Reply:
yeah, I know that classic for is better. Will change to ```for```